From d8ee8b82b7593c55644d3a6cf40ac19d7e7cac1f Mon Sep 17 00:00:00 2001
From: WRS Support <support@windriver.com>
Date: Fri, 2 Oct 2009 16:38:02 -0400
Subject: [PATCH 1/3] mpc8572: delete core1 from DTS (FIXME FIXME FIXME)

This should not be needed; can we boot UP even with a 2nd
entry in the dts?  If so, then delete this turd.

Signed-off-by: Dev Null <dev.null@windriver.com>
---
 arch/powerpc/boot/dts/mpc8572ds.dts |   13 -------------
 1 files changed, 0 insertions(+), 13 deletions(-)

diff --git a/arch/powerpc/boot/dts/mpc8572ds.dts b/arch/powerpc/boot/dts/mpc8572ds.dts
index 0a1fdd1..58b438d 100644
--- a/arch/powerpc/boot/dts/mpc8572ds.dts
+++ b/arch/powerpc/boot/dts/mpc8572ds.dts
@@ -43,19 +43,6 @@
 			clock-frequency = <0>;
 			next-level-cache = <&L2>;
 		};
-
-		PowerPC,8572@1 {
-			device_type = "cpu";
-			reg = <0x1>;
-			d-cache-line-size = <32>;	// 32 bytes
-			i-cache-line-size = <32>;	// 32 bytes
-			d-cache-size = <0x8000>;		// L1, 32K
-			i-cache-size = <0x8000>;		// L1, 32K
-			timebase-frequency = <0>;
-			bus-frequency = <0>;
-			clock-frequency = <0>;
-			next-level-cache = <&L2>;
-		};
 	};
 
 	memory {
-- 
1.6.3.3

